wrapper_test.go

```diff
@@ -390,5 +390,4 @@ func (w *wrapperTestSuite) TestCheckFailWriteSpanHandler_FailLoadConfig() {
 	assert.Equal(w.T(), 1, len(dirEntries))
 
 	assert.Equal(w.T(), "balagan_stop", dirEntries[0].Name())
-
 }
```

Comment:
I think we also need a test that verifies that function-span doesn't have the httpInfo.

Reply:
will do

Reply:
this happens here: internal/transform/span_test.go in TestTransform